docs/source/_static/dark.css

```diff
@@ -2411,15 +2412,15 @@
         a.wy-text-neutral:hover {
             color: rgb(177, 170, 160) !important;
         }
-        hr {
-            border-color: rgb(120, 112, 99) currentcolor currentcolor;
+        hr {  /* PTB (modified) - makes the top divider line in docs the same as background */
+            border-color: rgb(21, 22, 23) currentcolor currentcolor;
```

Comment:
Can you include the original values in a comment? might make search-and-replace easier when updating the theme …